cnvlib/coverage.py

```diff
@@ -104,15 +112,60 @@ def filter_read(read):
     return count, depth
 
 
-def interval_coverages_pileup(bed_fname, bam_fname, min_mapq):
+def rm(path):
+    try:
+        os.unlink(path)
+    except OSError:
+        pass
+
+def to_chunks(bed_fname, chunk_size=5000):
+    """Split the bed-file into chunks for parallelization"""
+
+    k, chunk = 0, 0
+    fd, name = tempfile.mkstemp(suffix=".bed", prefix="tmp.%s." % chunk)
```

Comment:
Since I'm returning the file name from the funciton, that exits the with scope. To use named-temporary file, we'd have to inline this into the calling function. What do you think?